tests/light/src/axosyslog_light/syslog_ng/syslog_ng_docker_executor.py

```diff
@@ -59,11 +61,18 @@ def run_process(
         command += ["--workdir", str(Path.cwd().absolute())]
         command += ["--user", f"{os.getuid()}:{os.getgid()}"]
         command += ["-e", f"PUID={os.getuid()}", "-e", f"PGID={os.getgid()}"]
-        command += ["--network", "host"]
+
 
         for path in paths:
             command += ["-v", f"{path}:{path}"]
 
+        if self.__extra_env_var:
+            command += ["-e", self.__extra_env_var]
+
+        if self.__extra_volume_mounts:
+            for mount in self.__extra_volume_mounts:
+                command += ["-v", mount]
```

Comment:
I think this should be a dict at least, so we can mount something from the host to a different place in the container. Maybe a list or set of 2 element tuples would be the best.